packages/perseus-editor/src/editor.tsx

```diff
@@ -826,11 +826,7 @@ class Editor extends React.Component<Props, State> {
         }
     };
 
-    serialize: (options?: any) => {
-        content: string;
-        images: any;
-        widgets: Record<any, any>;
-    } = (options: any) => {
+    serialize(options?: any): PerseusRenderer {
```

Comment:
Refactoring `serialize` to a method instead of an arrow function changes semantics - `this` might be unbound.